src/main/java/lotto/LottoRankEnum.java

```diff
@@ -0,0 +1,20 @@
+package lotto;
+
+public enum LottoRankEnum {
```

Comment:
```suggestion
public enum LottoRank {
```
클래스 네임에 enum을 굳이 명칭할 필요는 없을 것 같아요 😄